config/initializers/rack_attack.rb

```diff
@@ -17,32 +17,59 @@ class Rack::Attack
     "/api/docs",
   ].freeze
 
+  def self.api_request?(request)
+    request.path.starts_with?("/api/")
+  end
+
+  def self.public_api_path?(request)
+    PUBLIC_API_PATH_PREFIXES.any? { |prefix| request.path.starts_with?(prefix) }
+  end
+
+  def self.protected_path?(request)
```

Comment:
if these aren't reused methods do we need to move them out here 🤔

Reply:
we don't _have_ to, but I thought it made the rules easier to read 🤷‍♂️ happy to put them back if you feel differently